### ISSUE TITLE
Cookie consent window keeps appearing on local use (file:// protocol) with Firefox

### ISSUE THREAD

**Issue by wilhelmer** [squidfunk/mkdocs-material#3202]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

When accessing the output locally (file:// protocol) with Firefox (v94, Windows), the cookie consent window keeps reappearing on each page. 

The setting is stored for each page individually, so if you go to topic A, click Accept, and switch to topic B, you will have to click Accept again. If you then return to topic A, the consent window won't show on that page. But the setting isn't stored globally. 

I suggest to **never** display the cookie consent window at all on local use. I don't think the GDPR stuff applies to content that is on your own hard disk. Also, GA doesn't work anyway when the content is accessed via file protocol.

### Expected behaviour

The cookie consent window shouldn't reappear after switching pages, or never appear at all.

### Actual behaviour

Cookie consent window keeps reappearing on each page. 

### Steps to reproduce

1. Enable the cookie consent feature.
2. Open topic A. Cookie window appears.
3. Click "Accept".
4. Switch to topic B. Cookie window appears again.

### Package versions

- Python: 3.9.4
- MkDocs: mkdocs, version 1.2.3 
- Material: Version: 7.3.6+insiders.3.2.0

### Configuration

```yaml
site_name: My Docs
theme:
  name: material
use_directory_urls: false
extra:
    # Analytics and cookie consent
    analytics:
        provider: google
        property: UA-xxxxxx
    consent:
        title: This website uses cookies
        description: >
            We use cookies to recognize your repeated visits and preferences, as well
            as to measure the effectiveness of our documentation and whether users
            find what they're searching for. With your consent, you're helping us to
            make our documentation better.
```

### System information

- Operating system: Windows
- Browser: Firefox 94

**Comment by squidfunk** [comment]
Thanks for reporting

> I suggest to never display the cookie consent window at all on local use. I don't think the GDPR stuff applies to content that is on your own hard disk. Also, GA doesn't work anyway when the content is accessed via file protocol.

I agree, as we can't make XHR calls anyway, so there's nothing to be worried about.

**Comment by squidfunk** [comment]
I've tried reproducing the issue with Firefox, but I didn't manage to. For me, the consent goes aways after I accepted it. However, showing it during local browsing makes no sense. Fixed in [`da3540999`](https://github.com/squidfunk/mkdocs-material-insiders/commit/da3540999eb69a760b30a74402b1b0074d6a5173).

**Comment by wilhelmer** [comment]
Might be because I have strict mode enabled in Firefox. The strict mode settings don't show up during local browsing, so I thought it doesn't matter. But maybe it does. Anyway, thanks for fixing.

**Comment by squidfunk** [comment]
The fix was just released as part of 7.3.6-insiders-3.2.3.
